ios/PasskeyDelegate.swift

```diff
@@ -1,146 +1,156 @@
-import Foundation
 import AuthenticationServices
+// ! adapted from https://github.com/f-23/react-native-passkey/blob/fdcf7cf297debb247ada6317337767072158629c/ios/PasskeyDelegate.swift
+import Foundation
+
+class PasskeyDelegate: NSObject, ASAuthorizationControllerDelegate,
+    ASAuthorizationControllerPresentationContextProviding
+{
+    private let handler: PasskeyResultHandler
 
-class PassKeyDelegate: NSObject, ASAuthorizationControllerDelegate, ASAuthorizationControllerPresentationContextProviding  {
-  private let handler: PasskeyResultHandler
+    init(handler: PasskeyResultHandler) {
+        self.handler = handler
+    }
 
-  init(handler: PasskeyResultHandler) {
-    self.handler = handler
-  }
+    // Perform the authorization request for a given ASAuthorizationController instance
+    @available(iOS 15.0, *)
+    func performAuthForController(controller: ASAuthorizationController) {
+        controller.delegate = self
+        controller.presentationContextProvider = self
+        controller.performRequests()
+    }
 
-  // Perform the authorization request for a given ASAuthorizationController instance
-  @available(iOS 15.0, *)
-  func performAuthForController(controller: ASAuthorizationController) {
-    controller.delegate = self;
-    controller.presentationContextProvider = self;
-    controller.performRequests();
-  }
+    @available(iOS 13.0, *)
+    func presentationAnchor(for controller: ASAuthorizationController) -> ASPresentationAnchor {
+        guard let windowScene = UIApplication.shared.connectedScenes
+            .compactMap({ $0 as? UIWindowScene })
+            .first,
+            let window = windowScene.windows.first else {
+            return ASPresentationAnchor()
+        }
+        return window
+    }
 
-  @available(iOS 13.0, *)
-  func presentationAnchor(for controller: ASAuthorizationController) -> ASPresentationAnchor {
-    return UIApplication.shared.keyWindow ?? ASPresentationAnchor()
-  }
+    @available(iOS 13.0, *)
+    func authorizationController(
+        controller: ASAuthorizationController,
+        didCompleteWithError error: Error
+    ) {
+        handler.onFailure(error)
+    }
 
+    @available(iOS 13.4, *)
```

Comment:
i wonder why xcode doesn't allow remove this check if target is already ios 15

Reply:
removed in https://github.com/ExodusMovement/react-native-passkeys/pull/23/commits/8397f3218d716312d8c45d370a2a0c22b92098ba